spec/blueprints/sub_category_blueprint_spec.rb

```diff
@@ -7,15 +7,15 @@
     context "basic" do
       let(:name) { "new_sub_category" }
       let(:category) { create(:category) }
-      let(:sub_category) { create(:sub_category, name: name, category_id: category.id) }
-      let(:blueprint_sub_category) { SubCategoryBlueprint.render_as_hash(sub_category, view: :basic) }
+      let(:sub_category) { create(:sub_category, name: name, category: category) }
+      subject { described_class.render_as_hash(sub_category, view: :basic) }
 
       it "validates numbers of elements" do
-        expect(blueprint_sub_category.count).to eql(2)
+        expect(subject.count).to eql(3)
       end
 
       it "checks the content" do
-        expect(blueprint_sub_category).to include(name: name)
+        expect(subject).to include(name: name, category_id: category.id)
```

Comment:
`it "validates numbers of elements" do` - remove

one simple test
`expect(subject).to eq { id: ..., name: ..., category_id: ... }`